src/lxc/confile_utils.c

```diff
@@ -3,18 +3,19 @@
  * Copyright © 2017 Christian Brauner <christian.brauner@ubuntu.com>.
  * Copyright © 2017 Canonical Ltd.
  *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2, as
- * published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License along
- * with this program; if not, write to the Free Software Foundation, Inc.,
- * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
```

Comment:
Please get the following people to agree with this change:
 - 0x0916
 - 2xsec
 - Adam Borowski
 - Christian Brauner
 - Donghwa Jeong
 - Felix Abecassis
 - Josh Soref
 - LiFeng
 - Tycho Andersen

Reply:
@0x0916 @2xsec @kilobyte @flx42 @jsoref @lifeng68 @tych0

Reply:
Got ack from @jsoref in PR comment.

Reply:
Got ack from @tych0 in PR comment.

Reply:
✓

Reply:
I agree.

Reply:
I agree.

Reply:
@stgraber I agree.